scripts/experience-decisioning/preview.js

```diff
@@ -214,7 +214,7 @@ async function decorateExperimentPill(overlay) {
       label: config.label,
       description: `
         <div class="hlx-details">
-          ${config.status}${config.audience ? ', ' : ''}${config.audience}${config.variants[config.variantNames[0]].blocks.length ? ', Blocks: ' : ''}${config.variants[config.variantNames[0]].blocks.join(',')}
+          ${config.status}${config.audiences.length ? ', ' : ''}${config.audiences[0]}${config.variants[config.variantNames[0]].blocks.length ? ', Blocks: ' : ''}${config.variants[config.variantNames[0]].blocks.join(',')}
```

Comment:
Only the 1st one actually applies for now, so I think it's more explicit what happens if we only show the 1st one here as well, otherwise we may imply the experiment is running for another audience than the one resolved at that time

Reply:
> Only the 1st one actually applies for now, ...
uhmmmm, if I got it right the experiment runs if any of the audiences configured in the experiment is resolved. Not only if the first one is resolved.

Reply:
In the pill we are just informing the practitioner about the configured audiences for the experiment, we are not calculating and showing the "resolved audiences" for the current device of the practitioner. Or is the idea to show in the pill the resolved audiences, so practitioners can test with different devices/requests to validate the experiment? 

In any case, I think `config.audiences[0]` is just the first audience configured in the experiment, is not the first resolved one.

The layout stuff is an issue indeed. Maybe we could have the first audience and `...` with kind of a tooltip showing the rest.

Reply:
As I said below, we don't have any clear vision for the UI and what to expose in it, so for now I'd do the minimal approach and let customers ask for more if needed.

Agreed on the fact that this should show the resolved experiment and not the configured ones. Will fix this